sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/AccountDataResolver.scala

```diff
@@ -0,0 +1,5 @@
+package com.azure.cosmos.spark
+
+trait AccountDataResolver {
+    def getAccountDataConfig(linkedServiceName: Option[String]): Map[String, String]
```

Comment:
OLTP spark connector on github is not supposed to know about this config name.

the synapse specific is leaking to OLTP source code.

I would suggest we change this to something like
```scala
trait AccountDataResolver {
    def getAccountDataConfig(Map[String, String]): Map[String, String]
}
```

the input should be all configs (including synapse LinkedServiceName) and output should be all configs where synapse LinkedServiceName is replaced by account endpoint credentials.

Reply:
Resolved